packages/main/package.json

```diff
@@ -38,6 +38,6 @@
   },
   "devDependencies": {
     "@ui5/webcomponents-tools": "1.0.0-rc.12",
-    "chromedriver": "88.0.0"
+    "chromedriver": "^88.0.0"
```

Comment:
I think this change is not desired or?

Reply:
it was added when I downloaded the chromedriver, removed it :)